dmscripts/generate_framework_agreement_signature_pages.py

```diff
@@ -95,28 +114,79 @@ def render_html_for_suppliers_awaiting_countersignature(rows, framework, templat
         data['countersigned_at'] = datetime.strptime(
             data['countersigned_at'], '%Y-%m-%dT%H:%M:%S.%fZ'
         ).strftime('%d %B %Y')
+        if data['signed_agreement_returned_at']:
+            data['signed_agreement_returned_at'] = datetime.strptime(
+                data['signed_agreement_returned_at'], '%Y-%m-%dT%H:%M:%S.%fZ'
+            ).strftime('%d %B %Y')
         data['includeCountersignature'] = True
-        html = render_html(template_path, data)
-        save_page(html, data['supplier_id'], output_dir, "agreement-countersignature")
-    shutil.copyfile(template_css_path, os.path.join(output_dir, 'framework-agreement-signature-page.css'))
-    shutil.copyfile(
-        template_countersignature_path,
-        os.path.join(output_dir, 'framework-agreement-countersignature.png')
-    )
+
+        html_dir = output_dir / str(data['supplier_id'])
```

Comment:
So, we are changing the path from
`output_dir/<files>` to
`output_dir/1234/<files>`...
I think this is fine, because it's just a temporary location and we are passing the full path every time, as far as I can see, but we may want to do some broader testing on preview to check nothing breaks. since dmscripts is not part of functional/smoke tests.

Reply:
Agree, for now testing on preview/jenkins is the best way to validate this works as expected. Same with symlinking the files.